src/main/java/org/openstreetmap/atlas/tags/LevelTag.java

```diff
@@ -0,0 +1,48 @@
+package org.openstreetmap.atlas.tags;
+
+import java.util.Optional;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+import org.openstreetmap.atlas.tags.annotations.extraction.LongExtractor;
+
+/**
+ * OSM level tag
+ *
+ * @author sayas01
+ */
+@Tag(value = Tag.Validation.LONG, taginfo = "http://taginfo.openstreetmap.org/keys/level#values", osm = "http://wiki.openstreetmap.org/wiki/Level")
```

Comment:
I'm not sure that Long is the correct validator? From a brief look, around 5-6% of valid values are represented as strings. Thoughts on changing this to String? We could make it even more robust by accounting for split values and trying to convert to a Long right away, otherwise falling back to split on `;` for the correct value.

Reply:
The values for this tag are sometimes a combination of character and numeric, split by `,`,`;` or `:`.  Also there are values like `1-19` representing a range. There would be a couple of cases we will need to handle to get the correct value. For now, I have updated the type to String. Let me know your thoughts on this.